Library/Formula/libnfc.rb

```diff
@@ -31,14 +32,27 @@ def install
 end
 
 __END__
+diff --git a/libnfc/buses/uart.c b/libnfc/buses/uart.c
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!